src/Examine/LuceneEngine/Search/LuceneSearchQueryBase.cs

```diff
@@ -81,11 +84,16 @@ public IOrdering All()
         }
 
         /// <inheritdoc />
-        public IBooleanOperation NativeQuery(string query)
+        public IBooleanOperation NativeQuery(string query, ISet<string> loadedFieldNames = null)
```

Comment:
optional parameters are binary breaking changes. These will need to be overloads.